### PR TITLE
Copied the pre-rendering methods to articles, locations, and studies

### PR DESCRIPTION
**Pull request by codingwithlis**
### About
The pre-rendering methods (getStaticPaths and getStaticProps) were copied from Messages, Events, and title.js pages to articles, locations, studies pages.

### CODE REVIEW

File: pages/studies/[title].js [hunk 1]
```diff
@@ -24,6 +27,39 @@ export default function Study(props) {
   );
 }
 
+// This function gets called at build time to generate the titles for _all_ messages
+export async function getStaticPaths() {
+  // todo : make this a Network request so that it's dynamic
+  const titles = [];
+
+  return {
+    paths: titles.map(title => `locations/${title}`),
```
Comment (asleepingpanda) [inline pages/studies/[title].js:36]: Update `/locations` to `/studies`

File: pages/studies/[title].js [hunk 2]
```diff
@@ -24,6 +27,39 @@ export default function Study(props) {
...
+  };
+}
+
+// This also gets called at build time
+export async function getStaticProps({ params }) {
+  const apolloClient = initializeApollo();
+
+  await apolloClient.query({
+    query: GET_CONTENT_ITEM,
+    variables: { pathname: `locations/${params.title}` },
```
Comment (asleepingpanda) [inline pages/studies/[title].js:49]: Update `locations/` to be `studies/`

**Review by asleepingpanda — CHANGES_REQUESTED** [review]

File: pages/studies/[title].js (hunk 1 above)
Comment (codingwithlis) [reply pages/studies/[title].js]: Just fixed it, thank you!

File: pages/studies/[title].js (hunk 2 above)
Comment (codingwithlis) [reply pages/studies/[title].js]: Just fixed it, thank you!

**Review by asleepingpanda — CHANGES_REQUESTED** [review]
@codingwithlis there are a couple of old, unnecessary comments on some of these files. Could you please review them and remove the old comments.

**Pull request merged**